### DISCUSSION TITLE
Let the apps (e.g. iOS) be a sensor for manual gauges

### DISCUSSION THREAD

**Discussion by chrisb4711** [home-assistant/frontend#20947] [Other feature requests]
Dear all, 

yes it seems strange that I want analogue gauges to be integrated semi manual with my iPhone camera.
But the fact of my life is that I still have a lot analogue meters for water, ac, heating water temperature, pressure water, … 

so my basic idea is that you can use you phone take a photo from the gauge and send a signal to HA for a sensor. 

For longer measurement of energy that makes sense. Short term is too manual. 

Chris

**Comment by MindFreeze** [comment]
This sounds like a job for the new AI tasks. Check out this video https://www.youtube.com/watch?v=-bLVTHzfHyk
